Silence compile time warning -- mlx4/cq.c

This eliminates the compiler warning:
cq.c: In function `mlx4_ib_poll_cq':
cq.c:336: warning: 'srq' might be used uninitialized in this function

The code was correct; nothing was used unitialized, but it was too clever
for the 2.6.9 compiler to trace the logic.

Signed-off-by: John Gregor <john.gregor@qlogic.com>

---
 drivers/infiniband/hw/mlx4/cq.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: ofed_kernel/drivers/infiniband/hw/mlx4/cq.c
===================================================================
--- ofed_kernel.orig/drivers/infiniband/hw/mlx4/cq.c
+++ ofed_kernel/drivers/infiniband/hw/mlx4/cq.c
@@ -541,7 +541,7 @@ static int mlx4_ib_poll_one(struct mlx4_
 	struct mlx4_cqe *cqe;
 	struct mlx4_qp *mqp;
 	struct mlx4_ib_wq *wq;
-	struct mlx4_ib_srq *srq;
+	struct mlx4_ib_srq *srq = NULL;
 	struct mlx4_srq *msrq;
 	int is_send;
 	int is_error;
